### DISCUSSION TITLE
OpenMP parallelization in Open3D

### DISCUSSION THREAD

**Discussion by kimanton** [isl-org/Open3D#6844] [Q&A]
I have some questions regarding OpenMP in Open3D. I compiled Open3D from latest sources with OpenMP enabled, but can't see a difference when using ICP registration - htop still shows only one core loaded. 
Is there a particular functionality that benefits from OpenMP? Or do I need to do some additional setup in my code? 

I also have acados library with OpenMP enabled, and I can see all CPU cores loaded when doing nonlinear optimization.

**Comment by JaouadROS** [comment]
I have the same issue using python, only one thread is running at 100%. How can I make sure the parallelization is enabled?

**Comment by EngineerNitin** [comment]
@kimanton  Are you seeing this on Windows machines? I could observe significant difference earlier between Linux and WIndows, Linux being ~2X faster

**Reply by kimanton** [reply]
Only used it on Ubuntu
